packages/api/src/apps/serverless_read/app.ts

```diff
@@ -107,7 +108,21 @@ export default async (env: ProcessEnv) => {
     // TODO: switch this to root path once frontend is ready to transition
     ["global", Actions.Global(undefined, appState)],
   ];
+  // it looks like we want to enable tenderly simulations, so we are goign to validate env and enable osnap route
+  if (env.TENDERLY_ACCOUNT_NAME || env.TENDERLY_PROJECT_NAME || env.TENDERLY_ACCESS_KEY) {
+    assert(env.TENDERLY_ACCOUNT_NAME, `Osnap requires TENDERLY_ACCOUNT_NAME`);
+    assert(env.TENDERLY_PROJECT_NAME, `Osnap requires TENDERLY_PROJECT_NAME`);
+    assert(env.TENDERLY_ACCESS_KEY, `Osnap requires TENDERLY_ACCESS_KEY`);
+    const tenderlyConfig = {
+      accountName: env.TENDERLY_ACCOUNT_NAME,
+      projectName: env.TENDERLY_PROJECT_NAME,
+      accessKey: env.TENDERLY_ACCESS_KEY,
+    };
+    const tenderlies = new libs.osnap.MultiChainTenderly(tenderlyConfig);
+    channels.push(["osnap", Actions.Osnap({ tenderlies })]);
+    console.log("Enabled Tenderly Simulations for Osnap");
+  }
 
   await Express({ port: Number(env.EXPRESS_PORT), debug }, channels)();
-  console.log("Started Express Server, API accessible");
+  console.log("Started Express Server, API accessible on port", env.EXPRESS_PORT);
```

Comment:
Isn't it better to use `PORT` variable name if this is run in serverless as suggested in [docs](https://cloud.google.com/run/docs/developing)?

Reply:
thats interesting. wil consider that for the future, but since this app is already running with existing vars, i wouldnt risk changing this now